pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -105,6 +134,7 @@ func TestRESTGet(t *testing.T) {
 }
 
 func TestRESTDelete(t *testing.T) {
+	fakeClock := clocktesting.NewFakeClock(time.Now())
```

Comment:
follow the example I gave above: `now := time.Now()`

then in each *subtest*, create a separate clock with `fakeClock := clocktesting.NewFakeClock(now)`